### ISSUE TITLE
Validate Inputs use supported types

### ISSUE THREAD

**Issue by timsearle** [TWiStErRob/net.twisterrob.ghlint#301]
[Supported Types](https://docs.github.com/en/actions/learn-github-actions/contexts#inputs-context)

Invalid example:

```yaml
on:
  workflow_call:
    inputs:
      some-value:
        type: number
```

Valid example:

```yaml
on:
  workflow_call:
    inputs:
      some-value:
        type: string
```

**Comment by TWiStErRob** [comment]
JSON schema is (should be) doing this. Is it not?

**Comment by TWiStErRob** [comment]
Hmm, the docs have an example where number is used:
https://docs.github.com/en/actions/learn-github-actions/contexts#example-usage-of-the-inputs-context-in-a-reusable-workflow

What am I missing?

**Comment by timsearle** [comment]
PICNIC
